src/main/java/org/gridsuite/actions/server/ContingencyListService.java

```diff
@@ -89,12 +100,14 @@ List<ContingencyListAttributes> getContingencyLists() {
     }
 
     List<ContingencyListAttributes> getContingencyLists(List<UUID> ids) {
```

Comment:
Need to modify also `getContingencyLists() `

Reply:
Done